src/dotnet/APIView/APIViewWeb/Models/AssemblyModel.cs

```diff
@@ -0,0 +1,51 @@
+﻿using APIView;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace APIViewWeb.Models
+{
+    public class AssemblyModel
+    {
+        public AssemblyModel(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public int ID { get; set; }
```

Comment:
Nit - should this be cased as `Id`?

Reply:
I agree, I think it should be. The ASP.NET tutorials would sometimes alternate between `ID` and `Id` so I wasn't sure at the time. Fixed it in the code!